### DISCUSSION TITLE
build": "next build && next export "

### DISCUSSION THREAD

**Discussion by matheuskousen** [vercel/next.js#18922] [Help] [answered]
**"build": "next build && next export "**, doesn't it work anymore  on version 10 ?. How could i to gerate a build static ?

**Comment by jamesmosier** [comment, marked as answer]
Hi @matheuskousen. `next build && next export` is still how you can create a static HTML export Next app:

https://nextjs.org/docs/advanced-features/static-html-export

**Comment by onopko81** [comment]
hi! how export my project for insert in my server with a subdirectory? for example:
/experiment/my-project?

thanks

**Comment by onopko81** [comment]
basepath: '/directoryX' not working with image in public folder. how resolve it?
